src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/service/converter/ClientConverter.kt

```diff
@@ -23,14 +22,18 @@ class ClientConverter(
         clientSecretExpiresAt = null,
         clientName = clientIdService.toBase(clientId),
         clientAuthenticationMethods = CLIENT_SECRET_BASIC.value,
-        authorizationGrantTypes = CLIENT_CREDENTIALS.value,
+        authorizationGrantTypes = grantType.name.lowercase(),
```

Comment:
Can't you use the description field in GrantType here instead?

Reply:
Updated